x-pack/plugins/asset_manager/server/lib/accessors/services/get_services.ts

```diff
@@ -6,31 +6,42 @@
  */
 
 import { Asset } from '../../../../common/types_api';
-import { GetServicesOptionsInjected } from '.';
+import { GetServicesOptionsInjected } from './shared_types';
 import { collectServices } from '../../collectors/services';
+import { parseEan } from '../../parse_ean';
 
-export async function getServicesBySignals(
+export async function getServices(
   options: GetServicesOptionsInjected
 ): Promise<{ services: Asset[] }> {
   const filters = [];
 
   if (options.parent) {
-    filters.push({
-      bool: {
-        should: [
-          { term: { 'host.name': options.parent } },
-          { term: { 'host.hostname': options.parent } },
-        ],
-        minimum_should_match: 1,
-      },
-    });
+    const { kind, id } = parseEan(options.parent);
```

Comment:
should we validate `kind` value as part of `parseEan` (caller can pass the supported types to the function), or here, maybe with a switch ? thinking the caller should know if the parameter was ignored